src/Migrate/test/Initialize-AzMigrateHCIReplicationInfrastructure.Tests.ps1

```diff
@@ -0,0 +1,28 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Initialize-AzMigrateHCIReplicationInfrastructure'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Initialize-AzMigrateHCIReplicationInfrastructure.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Initialize-AzMigrateHCIReplicationInfrastructure' {
+    It 'Default' -skip {
```

Comment:
Please remove -skip tag for test cases and record them

Reply:
After removing the -skip tag to record, there is no Recording file that gets generated for some reason.

Reply:
As you added -Tag "LiveOnly" for this test case, please remove them if you want to record it.

Reply:
I added the Tag after seeing the recording didn't work for it